website/versioned_docs/version-stable/precommit.md

```diff
@@ -91,7 +91,7 @@ Alternately you can save this script as `.git/hooks/pre-commit` and give it exec
 
 ```bash
 #!/bin/sh
-FILES=$(git diff --cached --name-only --diff-filter=ACM "*.js" "*.jsx" | sed 's| |\\ |g')
+FILES=$(git diff --cached --name-only --diff-filter=ACMR "*.js" "*.jsx" | sed 's| |\\ |g')
```

Comment:
Good catch, I’ll update the stable docs so that this comes out.

Reply:
https://github.com/prettier/prettier/commit/8d0ac8bb58ccc0d30a47a000ca3adba7343cec90